int32 top
int32 bottom
int32 left
int32 right
int32 classID
int32 frameID
float64 propability
float64 processingTime
float64 timestamp
